pandas/tests/scalar/test_nat.py

```diff
@@ -66,6 +66,9 @@ def test_nat_vector_field_access():
         # on NaT/Timestamp for compat with datetime
         if field == "weekday":
             continue
+        if field in ["week", "weekofyear"]:
```

Comment:
we should properly parameterize these tests (if you would like to make an issue would be great), PR even better :->

Reply:
Im happy to make another PR, will wait until this is merged.